src/main/java/space/npstr/magma/connections/AudioWebSocket.java

```diff
@@ -198,16 +198,14 @@ private void handleReady(final Ready ready) {
     }
 
     private void handleSessionDescription(final SessionDescription sessionDescription) {
-        this.audioConnection.updateSecretKeyAndEncryptionMode(
-                sessionDescription.getSecretKey(),
-                sessionDescription.getEncryptionMode()
-        );
+        this.audioConnection.setSecretKey(sessionDescription.getSecretKey());
+        this.audioConnection.setEncryptionMode(sessionDescription.getEncryptionMode());
     }
 
     private void handleWebSocketClosed(final WebSocketClosed webSocketClosed) {
         final int code = webSocketClosed.getCode();
         log.info("Websocket to {} closed with code {} and reason {}",
-                wssEndpoint.toASCIIString(), code, webSocketClosed.getReason());
+                this.wssEndpoint.toASCIIString(), code, webSocketClosed.getReason());
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Invoke method(s) only conditionally. ([squid:S2629](https://sonarcloud.io/organizations/napstr-github/rules#rule_key=squid%3AS2629))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=space.npstr.magma%3Amagma&pullRequest=4&issues=AWQknaY8_bEzQ-kUd4C_&open=AWQknaY8_bEzQ-kUd4C_)

[ISSUE_KEY]: AWQknaY8_bEzQ-kUd4C_